<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>operateWD</class>
 <widget class="QWidget" name="operateWD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="bDeposit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>存钱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bWithdraw">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取钱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeDate">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>改变日期</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showMessage">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>550</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>显示流水</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="moneyEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="queryBrow">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="thingEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>金额</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>事件</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateBegin">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>490</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEnd">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>490</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>500</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>从</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>500</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>到</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
